src/DIRAC/Core/Utilities/ElasticSearchDB.py

```diff
@@ -265,9 +267,15 @@ def updateDoc(self, index: str, docID: str, body) -> dict:
         """
         sLog.debug(f"Updating document {docID} in index {index}")
         try:
-            return S_OK(self.client.update(index, docID, body))
+            self.client.update(index, docID, body)
+        except ConflictError:
+            # updates are rather "heavy" operations from ES point of view, needing seqNo to be updated.
+            # Not ideal, but we just wait and retry.
+            time.sleep(3)
+            self.client.update(index, docID, body, retry_on_conflict=3)
         except RequestError as re:
             return S_ERROR(re)
+        return S_OK()
```

Comment:
Did you intend to change the return value?

Reply:
Yes, anyway it is not used.